inc/object.h

```diff
@@ -65,4 +65,20 @@ typedef struct s_sphere
 	t_color	color;
 }	t_sphere;
 
+typedef struct s_plane
+{
+	t_vec3	pos;
+	float	vec;
```

Comment:
`vec`?

you are missing a normal
![image](https://github.com/user-attachments/assets/7feb09a2-eda6-4090-892f-f9a1cd853681)

```suggestion
	t_vec3	normal
```